src/dotnet/commands/dotnet-test/MessageHandlers/TestRunnerTestStartedMessageHandler.cs

```diff
@@ -14,8 +14,14 @@ public TestRunnerTestStartedMessageHandler(IReportingChannel adapterChannel)
 
         protected override bool CanHandleMessage(IDotnetTest dotnetTest, Message message)
         {
-            return dotnetTest.State == DotnetTestState.TestExecutionSentTestRunnerProcessStartInfo &&
+            return IsAtAnAcceptableState(dotnetTest) &&
```

Comment:
Hey, give me a suggestion? In my mind, it is an acceptable state for the message handler.